//public class Task2 {
// Если добавим массив на 9 эелементов, тогда сработает исключение ArithmeticException
//    public static void main(String[] args) {
//        int[] intArray = {1, 2, 3, 4, 6, 67, 9, 23, 5};
//        try {
//            int d = 0;
//            double catchedRes1 = intArray[8] / d;
//            System.out.println("catchedRes1 = " + catchedRes1);
//        } catch (ArithmeticException e) {
//            System.out.println("Catching exception: " + e);
//        }
//
//    }
//
//}
